lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,12 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
```

Comment:
Should this go in `#process_file` so you can use an implicit `begin`? I usually take explicit `begin`s as a smell pointing at ExtractMethod -- and you've already done that, so might as well take advantage.